src/utils/narrow.js

```diff
@@ -131,11 +131,26 @@ export const narrowFromMessage = (message: Message, email: string) => {
 
 export const extractTypeToAndSubjectFromNarrow = (
   narrow: Narrow,
-): ['private' | 'stream', string, string] => {
+): { type: 'private' | 'stream', display_recipient: string, subject: string } => {
```

Comment:
We use `this_casing` only for data coming from the API directly (we might even revisit that). This should be `displayRecipient` or just `recipient`.